src/hooks/index.js

```diff
@@ -123,6 +124,15 @@ function activate(agent) {
 
   logger = agent.logger;
 
+  // Plugin stuff
+  var api = new Plugin(agent);
+  Object.keys(agent.plugins).forEach(function(pluginName) {
+    if (toInstrument[pluginName]) {
```

Comment:
Good catch - in fact enough logic has changed that I'm going to add a new file specifically for plugins. Change pending